### DISCUSSION TITLE
Fluentd v1.15.0

### DISCUSSION THREAD

**Discussion by ashie** [fluent/fluentd#3803] [Announcement]
Fluentd v1.15.0 comes with 4 enhancements and 6 bug fixes. A huge thank to 4 contributors who made this release possible.

### Enhancement

* #3535 #3771 in_tail: Add log throttling in files based on group rules
* #3680 Add `dump` command to fluent-ctl
* #3712 Handle YAML configuration format on configuration file
* #3768 Add `restart_worker_interval` parameter in `<system>` directive to set
  interval to restart workers that has stopped for some reason.

### Bug fixes

* #3711 out_forward: Fix to update timeout of cached sockets
* #3754 in_tail: Fix a possible crash on file rotation when `follow_inodes true`
* #3755 output: Fix a possible crash of flush thread
* #3766 in_tail: Fix crash bugs on Ruby 3.1 on Windows
* #3774 in_tail: Fix a bug that in_tail cannot open non-ascii path on Windows
* #3782 Fix a bug that fluentd doesn't release its own log file even after rotated by external tools

### Misc

* #3489 in_tail: Simplify TargetInfo related code
* #3700 Fix a wrong issue number in CHANGELOG
* #3701 server helper: Add comments to linger_timeout behavior about Windows
* #3724 service_discovery: Fix typo
* #3745 #3753 #3767 #3783 #3784 #3785 #3787 test: Fix unstable tests and warnings

### Contributors to this release (Alphabetical order)

* Daijiro Fukuda
* Hiroshi Hatake
* JiHyunSong
* Takuro Ashie

<hr /><em>This discussion was created from the release <a href='https://github.com/fluent/fluentd/releases/tag/v1.15.0'>Fluentd v1.15.0</a>.</em>

**Comment by ali-ghasempor** [comment]
Hi,
Will the Docker image be available on DockerHub? (https://hub.docker.com/r/fluent/fluentd/)
